tools/serverpod_cli/test/analyzer/entities/entity_analyzer/field_persist_test.dart

```diff
@@ -210,6 +210,39 @@ void main() {
     },
   );
 
+  test(
+    'Given a class with a field with persist negated',
```

Comment:
chore: Don't think this is the right description for this test. Also missing the `when` and `then` part